libmscore/lyrics.cpp

```diff
@@ -260,12 +260,14 @@ void Lyrics::layout()
             }
 
       bool styleDidChange = false;
-      if ((_no & 1) && !_even) {
+      if (isEven() && !_even) {
```

Comment:
Hmm, now this does look confusing. One would need to look up what `isEven()` does to find that it does not work on `_even`, but on '_no' 
I know I siggested to use `isEven()`, but I change my mind in the light of how this looks now ;-)

Reply:
I feel like the real problem is `_even`. Should I change it to `_wasEven`?

Reply:
But then `void setEven(bool b)                            { _even = b; }` seems strange